src/main/java/com/example/demo/console/SpringBootConsoleApplication.java

```diff
@@ -0,0 +1,18 @@
+package com.example.demo.console;
+
+import com.example.demo.repo.JsonRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SpringBootConsoleApplication implements CommandLineRunner {
+
+    @Autowired
```

Comment:
Please, use constructor injection instead of field injection

Reply:
Yes, you are rigth. I change it.